docker-compose.yml

```diff
@@ -7,8 +7,12 @@ services:
     networks:
       - back-tier
   web:
-    image: markstory/lint-review
-    command: 'gunicorn -c /code/settings.py lintreview.web:app'
+    build: .
```

Comment:
For testing it locally this is useful, but I guess it is worth having a dev and prod'ish one, maybe?